hedera-node/src/main/java/com/hedera/services/ServicesState.java

```diff
@@ -116,6 +103,7 @@
 		static final int NUM_0120_CHILDREN = 10;
 		static final int NUM_0130_CHILDREN = 10;
 		static final int NUM_0140_CHILDREN = 10;
+		static final int NFTOKENS = 11;
```

Comment:
I think that you need initialize the nft tokens in `initialize(MerkleInternal previous)` similar to the previous (e.g. `scheduledTxs()` or `runningHashLeaf()`. Check the whole file where `scheduledTxs` has been added and similar to that add nfts.